recipes/ilamb/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set name = "ILAMB" %}
+{% set version = "2.3.1" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 15c2e0da0139d406df5883dad428ab8692aeebbb55368db0ed3b93466f9d19db
+
+build:
+  number: 0
+  skip: True  # [py3k]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - pip
+    - python =2
+    - setuptools
+  run:
+    - python =2
+    - cython
```

Comment:
Are `cython` and `proj4` a runtime requirement? They aren't listed in [the `setup.py`](https://bitbucket.org/ncollier/ilamb/src/5a63f22169dcb07b61c5b4c56250d7be9f8cc56c/setup.py?at=v2.3.1&fileviewer=file-view-default#setup.py-101:108).

Reply:
They aren't listed but helped resolve issues with older conda packages -- I don't think it's required now, so I'll remove them and see what happens with the CI builds.